### ISSUE TITLE
Schema viewer: null type not shown where format key used

### ISSUE THREAD

**Issue by duncandewhurst** [open-contracting/standard#498]
The schema viewer doesn't show ```null``` as an option for properties where the ```format``` key is used in the json schema, e.g. for ```uri``` or ```date-time```.

In the example below, ```null``` is a valid type for both ```maxExtentDate``` and ```durationInDays``` but it is only shown in the schema viewer for ```durationInDays```

### schema viewer

![image](https://user-images.githubusercontent.com/19265814/28312052-9cb045a6-6ba9-11e7-8ccc-b5ce6b1e47c3.png)

### json schema

```json
"maxExtentDate": {
          "description": "The period cannot be extended beyond this date. This field is optional, and can be used to express the maximum available data for extension or renewal of this period.",
          "format": "date-time",
          "title": "Maximum extent",
          "type": [
            "string",
            "null"
          ]
        },
        "durationInDays": {
          "description": "The maximum duration of this period in days. A user interface may wish to collect or display this data in months or years as appropriate, but should convert it into days when completing this field. This field can be used when exact dates are not known.  Where a startDate and endDate are given, this field is optional, and should reflect the difference between those two days. Where a startDate and maxExtentDate are given, this field is optional, and should reflect the diffence between startDate and maxExtentDate.",
          "title": "Duration (days)",
          "type": [
            "integer",
            "null"
          ]
        }
```

**Comment by Bjwebb** [comment]
Here's the place in docson.js that this happens - https://github.com/open-contracting/standard/blob/65f884d793f0b956163d7b6e54b99d4b5f887edf/standard/docs/en/_static/docson/docson.js#L148

My javascript is rather rusty. @kindly might be in a better place to change this.

**Comment by edugomez** [comment]
What do we expect to see here:

1 - _MaxExtentDate date-time, null_

or 

2 - _MaxExtentDate string, null (format: date-time)_

or something else?

The (expected) behaviour of ```docson.js``` is to replace ```type```(s) with ```format```

I think that 1 could be enough, as _date-time_ (or _uri_) is a subset of string (in this context, i.e. a type in a specific format). However, I'm not sure how clear is that for the whole audience of the docs.

**Comment by jpmckinney** [comment]
I think for clarity that we should pursue something like 2 which separates the type information from the  format information.

**Comment by jpmckinney** [comment]
closed by #608
